lib/Controller/MessagesController.php

```diff
@@ -443,7 +443,7 @@ private function attachmentIsImage(array $attachment): bool {
 	 * @return boolean
 	 */
 	private function attachmentIsCalendarEvent(array $attachment): bool {
-		return $attachment['mime'] === 'text/calendar';
+		return in_array($attachment['mime'],  ['text/calendar', 'application/ics'], true);
```

Comment:
Why `'application/ics'`?

Reply:
That's the mime set for some emails apparently

Reply:
```
Content-Type: application/ics; name=meetup.ics; method=PUBLISH; 
        charset=UTF-8
Content-Transfer-Encoding: base64
Content-Disposition: attachment; filename=meetup.ics
```

Reply:
https://gitlab.freedesktop.org/xdg/shared-mime-info/commit/9d73fc74d9386333de1ba49ffe7ba991ef498c37

I think this is fine :)